### ISSUE TITLE
Scribus 1.5.5 release, no GTK theme option

### ISSUE THREAD

**Issue by brittyazel** [flathub/net.scribus.Scribus#10]
The GTK theme option seems go be gone from Scribus with this new release, and a blank spot is left in it's place. Thus when booting into Scribus this morning I was set to use the fallback "Fusion" theme instead of the GTK theme.

This is unfortunate also, as I was really excited to make use of the new dark icon theme with a dark GTK theme

**Comment by jonatansteller** [comment]
I ran into a similar issue with a fallback window border and overall performance regression, but this happened several days before the update to 1.5.5, i.e. with the previous version 1.5.4 on Fedora 30. If I am not mistaken, the Telegram flatpack had the same issue at the same time, but was fixed by an update a day after I had noticed it. Maybe this is the commit that fixed it for Telegram?

https://github.com/flathub/org.telegram.desktop/commit/be6087d536c42048f9e1ad45979b8323dc7023a1
